### ISSUE TITLE
Sort order decision

### ISSUE THREAD

**Issue by brandonneil** [ITS-UofIowa/drupal-manifest#27]
Let's decide how the manifest should be sorted.

My initial thoughts were related to the following priorities:

1. Service (first grouped by standard or custom, this level can be also used in the future to group by development organization (clas, grad college, etc))

2. Sub domain (this way, all lab sites are together, org sites together, etc.

3. Alphabetical from there. Within a subdomain, the domains are alphabetized.

**Comment by richardbporter** [comment]
My only concern is relying on the 'type' key too much. Its really arbitrary at this point and only dictates whether a site has a Git repo or not. Should we sort by profile instead?

**Comment by brandonneil** [comment]
@richardbporter I wasn't thinking about this from a programmatic viewpoint where we would use the type key. My point was more about when/if we have other organizations using the manifest, we group them differently for convenience?

**Comment by richardbporter** [comment]
I agree that grouping is a good idea but I think we should sort by profile first then site, which is basically equivalent to what you're proposing.

Here is an example of a "custom" site that doesn't need a Git repo: https://github.com/ITS-UofIowa/researcherhandbook.research.uiowa.edu. Also, we could in theory have "standard" sites with Git repos.

Whatever we decide, we should try to automate because this list is going to get long!

**Comment by richardbporter** [comment]
Looks like we decided on part of this... :)

Sorted by application first, then what? Alphabetical by profile?

**Comment by genjohnson** [comment]
We just talked about this during our team meeting and decided to change the order of sites on uiowa701 (since they are a bit jumbled right now).

We decided to bring uiowa701 in line with the order of the other docroots by:
- listing uiowa.edu/* sites alphabetically first (beginning with the UI Homepage site)
- listing all other *.uiowa.edu sites alphabetically (regardless of site type)
- listing all non-uiowa.edu sites alphabetically
